src/content/reference/cluster-upgrades/index.md

```diff
@@ -0,0 +1,190 @@
+In order to keep all components up-to-date, to allow you to benefit from latest improvements, features and security fixes, we provide upgrades for the entire software stack in tenant clusters.
+
+At Giant Swarm we believe that frequent, small updates are a way to keep change in the system manageable.
+We invest a lot of effort in making this happen without disrupting your usage of your clusters.
+
+In this article, we explain how upgrades work in detail and how you should provide your workloads in order to keep them running during a tenant cluster upgrade and prevent disruptions of your applications.
+
+## Background and Concepts
+
+### The tenant cluster stack
+
+Among the third party components building a tenant cluster stack are
+
+- [CoreOS Container Linux](https://coreos.com/os/docs/latest/) as the node's operating system
+- [Docker](https://docs.docker.com/engine/) as a container runtime environment
+- [etcd](https://coreos.com/etcd/) as distributed storage for Kubernetes and Vault
+- [Kubernetes](https://kubernetes.io/) with its many sub-components
+- [Project Calico](https://www.projectcalico.org/) for virtual networking
+- [CoreDNS](https://coredns.io/) for cluster-internal name resolution
+- [Prometheus node exporter](https://github.com/prometheus/node_exporter) for hardware and OS metrics
+- [NGINX Ingress Controller](https://github.com/kubernetes/ingress-nginx) for connecting services with load balancers
+
+
+as well as many operators and controllers created and maintained by Giant Swarm.
+
+### Releases
+
+All of the items in the list above are released independent of each other by their vendors.
+At Giant Swarm we bundle specific versions of these components of the tenant cluster stack into a **release**.
+A release is identified by a [semantic version number](https://semver.org/).
+
+Once deployed, the tenant cluster stack is **immutable**.
+All components are deployed based on images, either of virtual machines or of Docker containers.
+No changes are ever made to components at runtime.
+This ensures that the stack running in your environment is the exact stack we have tested before.
+
+As a consequence, the only way to change the stack is to perform an upgrade, to switch to a new release.
+
+#### Release upgrade semantics
+
+As the semantic versioning system defines, an upgrade from one version to another is either one of these:
+
+- *Patch*: The smallest type of upgrade occurs when we publish bugfixes, security fixes, or make changes to the observability while maintaining the given functionality of the stack.
+This can include any sort of patch upgrades of third party components.
+
+- *Minor*: A minor upgrade occurs when we add functionality, while maintaining the functionality of the stack as it was in the previous version.
+This can include minor upgrades of third party components, with the exception of Kubernetes.
```

Comment:
What is the exception though?

Reply:
true, could make it clear that minor k8s for us counts as major (which we do mention somewhere else, but why not also here)

Reply:
That's resolved in the next sentence, so i wouldn't want to make it any more complicated.